File: app/src/at-vinhnguyen/java/asiantech/internship/summer/javacore/exercise7/Fibonacci.java
package asiantech.internship.summer.javacore.exercise7;

import android.util.Log;

import static asiantech.internship.summer.javacore.InputUtils.inputIntNumber;

public class Fibonacci {

    private static final String TAG = Fibonacci.class.getSimpleName();

    public static void main(String[] args) {
        int n = inputIntNumber(TAG);
        Log.d(TAG, "So fibonacci thu " + n + " la: " + getFibonacci(n));
    }

    private static int getFibonacci(int n) {
        if (n == 0 || n == 1) {
            return n;
        }
        return getFibonacci(n - 1) + getFibonacci(n - 2);
    }
}

File: app/src/at-vinhnguyen/java/asiantech/internship/summer/kotlincore/exercise7/Fibonacci.kt
package asiantech.internship.summer.kotlincore.exercise7

import asiantech.internship.summer.kotlincore.getFibonacciInPosition
import asiantech.internship.summer.kotlincore.inputIntNumber

object Fibonacci {

    @JvmStatic
    fun main(args: Array<String>) {
        print("Nhap n:")
        val n = inputIntNumber()
        println("So fibonacci thu " + n + " la: " + n.getFibonacciInPosition())
    }
}
